site/cypress/e2e/Cypress-Gradeable/hotkey.spec.js

```diff
@@ -47,4 +48,21 @@ describe('TA grading hotkey testing', () => {
         cy.get('body').type('{D}');
         cy.get('[data-testid="posts-list"]').should('contain', 'Discussion Posts');
     });
+
+    it('allows user to remove hotkey', () => {
+        cy.login();
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'details']);
+        cy.get('[data-testid="view-sections"]').click();
+        cy.get('[data-testid="grade-button"]').eq(12).click();
+        cy.get('[data-testid="grading-setting-btn"]').click();
+        cy.get('[data-testid="remap-unset-0"]').click();
+        cy.get('[data-testid="remap-0"]').should('contain', 'Unassigned');
+        cy.get('[data-testid="remove-all-hotkeys"]').click();
+        cy.get('[data-testid="remap-1"]').should('contain', 'Unassigned');
+        cy.get('[data-testid="remap-2"]').should('contain', 'Unassigned');
+        cy.get('[data-testid="restore-all-hotkeys"]').click();
+        cy.get('[data-testid="remap-0"]').should('not.contain', 'Unassigned');
```

Comment:
Is there a value that you should expect to be here, not just `not unassigned`?

Reply:
Ye your right that makes a lot of sense